larray/core/session.py

```diff
@@ -94,10 +94,12 @@ def __init__(self, *args, **kwargs):
             if isinstance(a0, str):
                 # assume a0 is a filename
                 self.load(a0)
+            elif hasattr(a0, 'items'):
+                for k, v in a0.items():
+                    self[k] = v
```

Comment:
Since PR #775, we could call the ``Session.update()`` method in the init method to avoid some kind of duplicate code.

Reply:
ok, will do